doc/testing-your-application.rst

```diff
@@ -109,10 +111,22 @@ Constant                Getter                    Default
 ``VARNISH_PORT``        ``getCachingProxyPort()`` ``6181``                                           port Varnish listens on
 ``VARNISH_MGMT_PORT``   ``getVarnishMgmtPort()``  ``6182``                                           Varnish management port
 ``VARNISH_CACHE_DIR``   ``getCacheDir()``         ``sys_get_temp_dir()`` + ``/foshttpcache-varnish`` directory to use for cache
-``VARNISH_VERSION``     ``getVarnishVersion()``   ``4``                                              installed varnish application version
```

Comment:
this was rather puzzling me with my varnish 3 locally. i think it does make sense to use an environment over a constant here.